dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -25,6 +25,18 @@ properties:
         mandatory and if not present it defaults to 1 which is the
         configuration at power-up.
 
+    power-mode:
+      type: int
+      required: false
+      default: 0
+      description: Specify the sensor power mode. Default is power-up configuration.
+      enum:
+        - 0 # Low Power M1
+        - 1 # Low Power M2
+        - 2 # Low Power M3
+        - 3 # Low Power M4
+        - 4 # High Performance
```

Comment:
Similar comments here about moving the nice strings describing each value to the description so it shows up in the bindings index.